src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -567,18 +567,18 @@ def launch_fluent(
                 kwargs.update(cwd=cwd)
             if topy:
                 if isinstance(topy, str):
-                    journal_name = topy.split(".")[0]
-                    journal_ext = topy.split(".")[1]
-                    launch_string += f' -i {journal_name}.{journal_ext} -command="(api-start-python-journal \\\"\\\"{journal_name}.py\\\"\\\")"'  # noqa: E501
+                    name = topy.split(".")[0]
+                    ext = topy.split(".")[1]
+                    launch_string += f' -i {name}.{ext} -command="(api-start-python-journal \\\"\\\"{name}.py\\\"\\\")"'  # noqa: E501
                 if isinstance(topy, list):
                     all_scm_journal_names = ""
                     all_py_journal_names = ""
                     name_list = []
                     for journals in topy:
-                        journal_name = journals.rsplit(".")[0]
-                        journal_ext = journals.rsplit(".")[1]
-                        name_list.append(journal_name)
-                        all_scm_journal_names += f' -i {journal_name}.{journal_ext}'
+                        name = Path(journals).stem.split('.')[0]
```

Comment:
Why is _journals_ pluralised?

Reply:
Because it's list of journals.

Reply:
`topy` is the list that we are iterating over. journals is one element in that list - a string, which is the name or path of a journal.

Reply:
Removed plurals. Thank you.